ow_gazebo_plugins/src/OWLightControlPlugin/OWLightControlPlugin.cpp

```diff
@@ -0,0 +1,82 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "OWLightControlPlugin.h"
+#include <gazebo/rendering/rendering.hh>
+
+
+namespace gazebo
+{
+
+GZ_REGISTER_VISUAL_PLUGIN(OWLightControlPlugin)
+
+OWLightControlPlugin::OWLightControlPlugin() :
+  VisualPlugin()
+{
+}
+
+OWLightControlPlugin::~OWLightControlPlugin()
+{
+}
+
+void OWLightControlPlugin::Load(gazebo::rendering::VisualPtr visual, sdf::ElementPtr sdf)
+{
+  m_nodeHandle.reset(new ros::NodeHandle("OWLightControlPlugin"));
+
+  m_lightName.push_back("lander::lander_lights_link::lander_light_light0");
+  m_paramName.push_back("left_light");
+  m_lightIntensity.push_back(1.0);
```

Comment:
[optional] Could be a member function.

Reply:
Could be. If I did it more than twice I would probably make it a function.